sos/report/plugins/clear_containers.py

```diff
@@ -73,9 +77,9 @@ def setup(self):
             self.limit = None
 
         self.add_cmd_output('{} cc-env'.format(self.runtime))
-        self.attach_cc_config_files()
+        self.collect_cc_config_files()
```

Comment:
There is a merge conflict with https://github.com/sosreport/sos/commit/57d21346b2952bf7c995eaeec4a98e1ae77c6308 .

Reply:
Ack, fixed